src/lib.rs

```diff
@@ -3,6 +3,7 @@
 //!
 
 use std::ptr;
+use elements::bitcoin;
 
 use bitcoin::secp256k1;
```

Comment:
This should also be updated to `use elements::bitcoin::secp256k1`

Reply:
Done!